docs/intermediate-modding/custom-config-syncing.md

```diff
@@ -0,0 +1,210 @@
+
+---
+prev: false
+next: false
+description: An intermediate overview of how to sync custom configs for your Lethal Company mods.
+---
+
+# Custom Config Syncing
+
+::: warning
+**This is an intermediate article. While this introduces some C# concepts, it is recommended to understand C# and the basics of modding this game <i>before</i> reading this article.**
+:::
+
+## Preface
+A very common case for many mod developers is wanting to synchronize the host's configuration file with all other players.
+
+There are many different ways you could achieve this, but we will only go through the two most straightforward approaches that should work for most cases.
```

Comment:
As far as I can tell there's only one approach (i think the other one was removed)